ubg/src/pages/Room.js

```diff
@@ -267,6 +282,20 @@ function Room({setUsersData, setCurrentUser, setRoomData}) {
     });
   }
 
+  /**
+   * Sets up player to be able to play again
+   */
+  function playAgain() {
+    usersCollection.doc(user.uid).update({
+      alive: true,
+      chose: false,
+      order: Math.floor(Math.random() * 20),
```

Comment:
What are we using order for?

Reply:
Per game, everyone is given a random number (order). When the game starts, we assign roles based on what number they received.

Reply:
Cool cool cool